src/instance/commands/triggers/bits.ts

```diff
@@ -0,0 +1,94 @@
+  JUMBO_BACKPACK_UPGRADE: { bitValue: 4000, prettyName: 'Jumbo Backpack Upgrade (4k Bits)' },
+  MINION_STORAGE_EXPANDER: { bitValue: 1500, prettyName: 'Minion Storage Expander (1.5k Bits)' },
+  POCKET_SACK_IN_A_SACK: { bitValue: 8000, prettyName: 'Pocket Sack (8k Bits)' },
+  TRIO_CONTACTS_ADDON: { bitValue: 6450, prettyName: 'Abi Contacts (6.45k Bits)' },
+  ABICASE_SUMSUNG_1: { bitValue: 15_000, prettyName: 'Sumsung© G3 Case (15k Bits)' },
+  ABICASE_SUMSUNG_2: { bitValue: 25_000, prettyName: 'Sumsung© GG Case (25k Bits)' },
+  ABICASE_REZAR: { bitValue: 26_000, prettyName: 'Rezar® Case (26k Bits)' },
+  ABICASE_BLUE_RED: { bitValue: 17_000, prettyName: 'Blue™ but Red Case (17k Bits)' },
+  ABICASE_BLUE_BLUE: { bitValue: 17_000, prettyName: 'Actually Blue™ Case (17k Bits)' },
+  ABICASE_BLUE_GREEN: { bitValue: 17_000, prettyName: 'Blue™ but Green Case (17k Bits)' },
+  ABICASE_BLUE_YELLOW: { bitValue: 17_000, prettyName: 'Blue™ but Yellow Case (17k Bits)' },
+  ABICASE_BLUE_AQUA: { bitValue: 17_000, prettyName: 'Lighter Blue™ Case (17k Bits)' },
+  DYE_PURE_WHITE: { bitValue: 250_000, prettyName: 'Pure White Dye (250k Bits)' },
+  DYE_PURE_BLACK: { bitValue: 250_000, prettyName: 'Pure Black Dye (250k Bits)' },
+  ENCHANTMENT_EXPERTISE_1: { bitValue: 4000, prettyName: 'Expertise (4k Bits)' },
+  ENCHANTMENT_COMPACT_1: { bitValue: 4000, prettyName: 'Compact Ench (4k Bits)' },
+  ENCHANTMENT_CULTIVATING_1: { bitValue: 4000, prettyName: 'Cult Ench (4k Bits)' },
+  ENCHANTMENT_CHAMPION_1: { bitValue: 4000, prettyName: 'Champ Ench (4k Bits)' },
+  ENCHANTMENT_HECATOMB_1: { bitValue: 6000, prettyName: 'Hecatomb Ench (6k Bits)' },
+  TALISMAN_ENRICHMENT_WALK_SPEED: { bitValue: 5000, prettyName: 'Spd Enrich (5k Bits)' },
+  TALISMAN_ENRICHMENT_INTELLIGENCE: { bitValue: 5000, prettyName: 'Int Enrich (5k Bits)' },
+  TALISMAN_ENRICHMENT_CRITICAL_DAMAGE: { bitValue: 5000, prettyName: 'Crit Dmg Enrich (5k Bits)' },
+  TALISMAN_ENRICHMENT_CRITICAL_CHANCE: { bitValue: 5000, prettyName: 'Crit Chance Enrich (5k Bits)' },
+  TALISMAN_ENRICHMENT_STRENGTH: { bitValue: 5000, prettyName: 'Str Enrich (5k Bits)' },
+  TALISMAN_ENRICHMENT_DEFENSE: { bitValue: 5000, prettyName: 'Def Enrich (5k Bits)' },
+  TALISMAN_ENRICHMENT_HEALTH: { bitValue: 5000, prettyName: 'Health Enrich (5k Bits)' },
+  TALISMAN_ENRICHMENT_MAGIC_FIND: { bitValue: 5000, prettyName: 'MF Enrich (5k Bits)' },
+  TALISMAN_ENRICHMENT_FEROCITY: { bitValue: 5000, prettyName: 'Fero Enrich (5k Bits)' },
+  TALISMAN_ENRICHMENT_SEA_CREATURE_CHANCE: { bitValue: 5000, prettyName: 'SC Chance Enrich (5k Bits)' },
+  TALISMAN_ENRICHMENT_ATTACK_SPEED: { bitValue: 5000, prettyName: 'Atk Spd Enrich (5k Bits)' },
+  TALISMAN_ENRICHMENT_SWAPPER: { bitValue: 200, prettyName: 'Acc Enrich Swapper (200 Bits)' }
+}
+
+export default class Bits extends ChatCommandHandler {
+  constructor() {
+    super({
+      name: 'Bits',
+      triggers: ['bits'],
+      description: 'Returns the best price',
+      example: `!bits`
```

Comment:
Remove prefix. It can't be hardcoded. Prefix is configurable. The commands manager take care of adding the prefix when displaying the message.

Reply:
How do you want this to be done?

Reply:
```suggestion
      example: `bits`
```

Reply:
Oh never mind, done.